css/util/color/index.css

```diff
@@ -0,0 +1,41 @@
+
+.bg-primary {
+    background-color: var(--primary-color)  !important;
+}
```

Comment:
In this complete file avoid using important, that should be the last preference.